source/includes/administration/_commitments.md

```diff
@@ -404,4 +404,22 @@ Object: **Product**
 
 Attributes | &nbsp;
 --- | ---
-`id`<br/>*UUID* | The id of the product.
\ No newline at end of file
+`id`<br/>*UUID* | The id of the product.
+
+
+<!-------------------- DELETE COMMITMENT -------------------->
+### Delete commitment
+
+`DELETE /commitments/:id`
+
+Deletes a commitment that has already been created for an organization.
+
+Can only be deleted if a commitment has no closed billing cycles associated to it. This means that a commitment will likely only be able to be deleted within a month of creation at most.
+
```

Comment:
We are missing the shell portion of the docs:

```shell
# Retrieve organization commitment list
curl "https://cloudmc_endpoint/api/v2/commitments" \
   -H "MC-Api-Key: your_api_key"
```

Reply:
Yes it isn't there but doesn't seem to be there for any of the other operations either

Reply:
I think we should add it for consistency, because most other endpoints have it no ?

Reply:
Yup large majority do, will follow up with the commitments one missing

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/568/commits/fce53d83c298ed74f139e832f3666aa988a91a12 shell script added in this commit